Set/Game/GameViewModel.swift

```diff
@@ -43,8 +100,20 @@ class GameViewModel: ObservableObject {
         let matchStatus = gameModel.toggleCard(by: chousenCard.id)
         if matchStatus == .successfulMatch || matchStatus == .unsuccessfulMatch {
             DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) { [weak self] in
-                self?.gameModel.finishTurn(for: matchStatus)
+                self?.gameModel.finishTurn(for: matchStatus, player: player)
+                self?.cleanUp()
             }
         }
     }
+    
+    private func cleanUp() {
+        whoseTurn = nil
+        timer?.invalidate()
+        timerTitle = ""
+        countDownTimer = 10
```

Comment:
Please rename it, cause it's not Timer, it's `countdown` (as one word, no camel case required`

Reply:
fixed